src/wlapplication.cc

```diff
@@ -1176,6 +1252,18 @@ void WLApplication::handle_commandline_parameters() {
 		commandline_.erase("replay");
 	}
 
+	if (commandline_.count("new_from_template")) {
```

Comment:
Please rename this command line option to make it more clear for what it stand for. E.g.:

"new_game_from_template"
"new_game_from_file"
"new_sp_game"
"load_game_from_file"